openproblems/tasks/_batch_integration/batch_integration_feature/api.py

```diff
@@ -11,7 +12,9 @@ def check_method(adata, is_baseline=False):
     return True
 
 
-sample_dataset = api.sample_dataset
+@dataset()
+def sample_dataset():
+    return api.sample_dataset(run_pca=True)
```

Comment:
`check_method` doesn't expect PCA, why is the sample method running PCA?

Reply:
we need the original embedding for the embedding metrics in the feature task. will add to check method and the README